### PR TITLE
Add support for RHEL based distributions

### PR DESCRIPTION
**Pull request by FlorentFlament**

### CODE REVIEW

File: recipes/_debian_scheduler.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  path '/etc/default/aurora-scheduler'
+  variables lazy { node['aurora']['scheduler'] }
+  user 'root'
+  group 'root'
+  mode '0644'
+  notifies :restart, 'service[aurora-scheduler]'
+end
+
+service 'aurora-scheduler' do
+  service_name 'aurora-scheduler'
```
Comment (jmauro) [inline recipes/_debian_scheduler.rb]: OPT: No need for this property.

File: recipes/_rhel_scheduler.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Do some stuff RHEL specific
+include_recipe 'java' if node['aurora']['install_java']
+
+# Add Aurora repository
+include_recipe "aurora::_rhel_repo"
```
Comment (jmauro) [inline recipes/_rhel_scheduler.rb]: I don't understand why you are installing mesos without using the mesos cookbook?

File: recipes/_rhel_slave.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  mode '0755'
+end
+yum_package 'aurora-executor-0.12.0-1' do
+  source 'aurora-executor-0.12.0-1.el7.centos.aurora.x86_64.rpm'
+  action :install
+end
+
+# Start thermos observer
+service 'thermos-observer' do
+  service_name 'thermos-observer'
```
Comment (jmauro) [inline recipes/_rhel_slave.rb]: OPT: property not needed

File: recipes/_debian_scheduler.rb (hunk 1 above)
Comment (FlorentFlament) [inline recipes/_debian_scheduler.rb]: That was for symmetry with `_rhel_scheduler.rb`, though I can remove this.

File: recipes/_rhel_slave.rb (hunk 3 above)
Comment (FlorentFlament) [inline recipes/_rhel_slave.rb]: That was for symmetry with `_debian_slave.rb`, though I can remove this.

File: recipes/_rhel_scheduler.rb (hunk 2 above)
Comment (FlorentFlament) [inline recipes/_rhel_scheduler.rb]: Indeed, this is useless. I'm fixing this.

**Comment by pierrecdn** [comment]
OPT: You should introduce and manage a version attribute. It could help to upgrade easily, even if the options args has to be reviewed at each upgrade.

**Comment by FlorentFlament** [comment]
@pierrecdn Fair enough, I'll update the cookbook for this.

**Comment by pierrecdn** [comment]
+1

**Pull request merged**